src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1080,8 +1080,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
 {
     protocol::TMProposeSet& set = *m;
 
-    if (overlay_.setup().expire &&
-            set.has_hops() && ! slot_->cluster())
+    if (set.has_hops() && ! slot_->cluster())
```

Comment:
I'm really glad the other two reviewers note this change is obviously correct.  I can't tell.  I looked in the README.md.  I looked at the `Setup::expire` declaration: https://github.com/ripple/rippled/blob/develop/src/ripple/overlay/Overlay.h#L70.  This seems to be the only place where it can get set `true`: https://github.com/ripple/rippled/blob/develop/src/ripple/overlay/impl/OverlayImpl.cpp#L890.  But this, too, has no clue about the *meaning* of the field.  I can't figure out what `overlay_.setup().expire()` meant.  Given that, I guess I'm glad the test is gone.

It may be worth adding some documentation in another commit.

Beyond that, the C++ looks fine to my eyes.